src/vm/wren_vm.c

```diff
@@ -1621,3 +1622,46 @@ void wrenGetVariable(WrenVM* vm, const char* module, const char* name,
   
   setSlot(vm, slot, moduleObj->variables.data[variableSlot]);
 }
+
+// Prints to WrenVM.errorFn if provided, otherwise prints to stderr.
+void wrenPrintError(WrenVM* vm, const char* format, ...)
+{
+  va_list args;
+  
+#if defined( _MSC_VER )
+  // MSVC doesn't have snprintf():
+  #define WREN_STRING_SIZE(format, args) _vscprintf(format, args)
+  #define WREN_STRING_PRINT(buffer, size, format, args) _vsnprintf_s(buffer, size, _TRUNCATE, format, args)
+#else
+  // Use C99's snprintf():
+  #define WREN_STRING_SIZE(format, args) vsnprintf(NULL, 0, format, args)
+  #define WREN_STRING_PRINT(buffer, size, format, args) vsnprintf(buffer, size, format, args)
+#endif
+
+  // Allocate `buffer` to hold the output:
+  va_start(args, format);
+  int sizeNeeded = WREN_STRING_SIZE(format, args);
+  int sizeAllocated = sizeNeeded + 1; // +1 for the null terminator
+  char *buffer = (char *)malloc(sizeAllocated);
+  va_end(args);
+
+  // Write the format and args into `buffer`:
+  va_start(args, format);
+  WREN_STRING_PRINT(buffer, sizeAllocated, format, args);
+  va_end(args);
+  
+  #undef WREN_STRING_SIZE
+  #undef WREN_STRING_PRINT
+  
+  // Output `buffer`:
+  if (vm->config.errorFn != NULL)
+  {
+    vm->config.errorFn(vm, buffer);
+  }
+  else
+  {
+    fprintf(stderr, "%s", buffer);
```

Comment:
Could you elaborate a bit on your thinking here? I ask because I'm not the biggest fan of adding a function for this (either to wren_vm.c, or perhaps wren_utils.h/.c); it seems like a natural behavior for `wrenPrintError()`. On the other hand, doing this would make it easier for the user to silence errors by setting it to `NULL`, assuming the check remains, minus the `else { fprintf(stderr, ...) }`.